mcs/class/corlib/System.Threading/Thread.cs

```diff
@@ -90,11 +90,12 @@ sealed class InternalThread : CriticalFinalizerObject {
 		private IntPtr interrupt_on_stop;
 		private IntPtr flags;
 		private IntPtr thread_pinning_ref;
+		private int priority;
 		/* 
 		 * These fields are used to avoid having to increment corlib versions
 		 * when a new field is added to the unmanaged MonoThread structure.
 		 */
-		private IntPtr unused1;
+		private int unused1;
```

Comment:
I think on 32-bit platforms (where IntPtr is 32-bits) this increases the object size by 4 bytes.